src/controllers/itemDetails/index.js

```diff
@@ -669,7 +669,7 @@ function reloadFromItem(instance, page, params, item, user) {
             location = `<a is="emby-linkbutton" class="button-link textlink" target="_blank" href="https://www.openstreetmap.org/search?query=${encodeURIComponent(location)}">${location}</a>`;
```

Comment:
Location should be escaped here to prevent an XSS injection.

Reply:
should be fixed with commit 55ad352d08e4162d1e99114b044176545f012501